### PR TITLE
Forms Automated Test - LPS-130999 | master

### PR DESCRIPTION
**Pull request by leilanyaragao**

### CODE REVIEW

**Comment by brittneyq** [comment]
Hi @leilanyaragao, since you didn't make any changes within the Button.path file, can you please remove that from your commit? Thank you!

**Comment by brittneyq** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/FormFields.path [hunk 1]
```diff
@@ -338,6 +338,27 @@
 	<td></td>
 	<td></td>
 </tr>
+<!--PAGE-->
```
Comment (diogocordeiro) [inline portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/FormFields.path:341]: Hi @leilanyaragao , I think that all these paths are not related to a Form Field. I believe that this path is more related to a Form feature, so please could you remove it from this file and change it to the Form.path file, please?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase [hunk 2]
```diff
@@ -2337,13 +2337,109 @@ definition {
 	}
 
 	@description = "Verify that Form Pages can be reordered"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test PagesCanBeReordered {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-130999 PagesCanBeReordered pending implementation
```
Comment (diogocordeiro) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase:2346]: After change the path remember to edit all places that uses this path

**Review by diogocordeiro — CHANGES_REQUESTED** [review]

**Comment by brittneyq** [comment]
ci:test:sf

**Comment by brittneyq** [comment]
ci:forward

**Pull request closed without merging**